src/api/java/com/minecolonies/api/research/ResearchBranchType.java

```diff
@@ -0,0 +1,43 @@
+package com.minecolonies.api.research;
+
+/**
+ * Different Research Branch types, along with descriptions of their behaviors.  Primarily related to display rendering internal to Minecolonies,
+ * or to implementations for AbstractResearchProvider.
+ */
+public enum ResearchBranchType
+{
+    /**
+     * Default branch using Tree-based view.  Enforces University Level, limits to one level-six research, shows research level headers, etc.
+     * All branches without an explicit type, or with an invalid explicit type, will act as DEFAULT.
+     */
+    DEFAULT("default"),
+    /**
+     * Achievement-style branch using Tree-based view.  Does not enforce university level, allows multiple researches above the university's max level,
+     * does not show research level requirement in headers, does not show research level time headers if branch-time is under 0.01.
+     * This is not a replacement for vanilla advancements: use only where University behaviors are important (eg: time, item cost, parent requirement enforcement),
+     * where requirement complexity is too broad to fit the Advancement system, where exclusive branches or colony-wide announcements are required,
+     * or where requirement or reward is tied to a colony-wide behavior instead of a player-specific one.
```

Comment:
I'm not happy with calling this "achievement" still. Can we make this maybe "unrestricted" or so?

Reply:
"unrestricted-tree" ok?  I'm kinda expecting to eventually find need for other layouts using the unrestricted behaviors.

Reply:
Could call it Unlockables, i think that's what Mekle is doing in his PR

Reply:
I prefer Achievements tbh. I feel like that tells players its purpose the fastest, and imo Unrestricted isn't very player-facing

Reply:
Disagree, Achievements has a reserved meaning, this is not 'achievement'

Reply:
Unlockables, Features, Bonuses, Specials.... it might be easier to pick a name if there was more explanation of possible intended uses.  I know that Mekle was intending to use this for unlocking mesh crafting in other huts based on the sifter hut level, are there some other examples?

Reply:
"Achievements" don't affect anything, so calling it achievements is kinda off. "Unlockables" "Bonuses" "Specials" would all work. For now it's only the mesh thing that will use it so far. So that could be "Feature" or "Bonus" or "Special" but, I think we fare better with "Unlockables" because it's more generic

Reply:
"Unlockable" is kinda awkward since it could conceivably not have any ResearchEffects on it at all (either by intent, by using research-ids for recipe unlocks, or if some external mod only uses research-ids).  This trait doesn't require or really have much impact on effects; it's focused on removing general restrictions for starting or displaying research.  But it works enough, so have swapped from "unrestricted-tree" to it. 

The thought is that this would be useful for: 
* potential multiplayer-focused Expert-style modpacks like Enigmatica or Magiculture, where a large number of potential huts and a variety of recipes would be blocked by requirement that certain building or item requirements were completed, including key components of entry to other mods.
* a guided-tour style focus, such as in Harvest Crossing or early phases of skyblocks, where the researches aren't having much actual effect, but the requirements to achieve them points you toward the underlying gameplay loop for that stage of the game.
* A research branch where research has both positive and negative effects (more practical once the more complicated effect system is in place).  Ie, a chain of research might increase archer damage but decrease their movement speed, with magnitude increasing each level, and it be intended that the colony owner select or even change this as the colony undergoes different demands (possibly even outside ones, like some modpacks that change spawn rules or AI for monsters over time, like Manufactio).

And then there could potentially be variants using a scrollinglist or whatever implementation is used for the guidebook, but otherwise shared most of the logic, for cases that needed more text space or intended a more linear presentation.

This particular string is also specifically json- and code-side so the only time users should see it is if working on a datapack or with the api.